lib/net/validations.rb

```diff
@@ -9,6 +12,10 @@ module Validations
     class Error < RuntimeError
     end
 
+    class << self
+      include Net::Validations
```

Comment:
I believe, this is not necessary at all. I changed the methods to just use "def self.abc". I think, that's the best approach.